tests/test_tti.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+import pytest
+from numpy import linalg
+
+from devito.logger import log
+
+from examples.acoustic.Acoustic_codegen import Acoustic_cg
+from examples.containers import IGrid, IShot
+from examples.tti.TTI_codegen import TTI_cg
+
+
+class TestTTI(object):
```

Comment:
Ok, I know this is how we do it in the acoustic test, but I see absolutely no reason to wrap this test in a class, since we are not storing anything on the object. The only thing it does is force us to use the `request.params` notation instead of setting individual parameter values explicitly like this:
```
@pytest.mark.parametrize('dimension', [(120, 140), (120, 140, 150)])
@pytest.mark.parametrize('space_order', [4])
def test_tti(dimension, space_order):
    ...
```